dao/src/test/java/org/thingsboard/server/dao/service/EdgeEventServiceTest.java

```diff
@@ -103,34 +100,31 @@ protected EdgeEvent generateEdgeEvent(TenantId tenantId, EdgeId edgeId, EntityId
     }
 
     @Test
-    public void findEdgeEventsByTimeDescOrder() throws Exception {
+    public void findEdgeEventsBySeqIdOrder_createdTimeOrderIgnored() throws Exception {
         EdgeId edgeId = new EdgeId(Uuids.timeBased());
         DeviceId deviceId = new DeviceId(Uuids.timeBased());
 
-        List<ListenableFuture<Void>> futures = new ArrayList<>();
-        futures.add(saveEdgeEventWithProvidedTime(timeBeforeStartTime, edgeId, deviceId, tenantId));
-        futures.add(saveEdgeEventWithProvidedTime(eventTime, edgeId, deviceId, tenantId));
-        futures.add(saveEdgeEventWithProvidedTime(eventTime + 1, edgeId, deviceId, tenantId));
-        futures.add(saveEdgeEventWithProvidedTime(eventTime + 2, edgeId, deviceId, tenantId));
-        futures.add(saveEdgeEventWithProvidedTime(timeAfterEndTime, edgeId, deviceId, tenantId));
-
-        Futures.allAsList(futures).get();
+        saveEdgeEventWithProvidedTime(timeBeforeStartTime, edgeId, deviceId, tenantId).get();
+        saveEdgeEventWithProvidedTime(eventTime, edgeId, deviceId, tenantId).get();
+        saveEdgeEventWithProvidedTime(eventTime - 2, edgeId, deviceId, tenantId).get();
```

Comment:
I suppose +2 is 'more clear' test case. 
You shows that you store eventTime, eventTime with +2, and later on you store eventTime+1.
In this case if order is working by created time, you would get 
`eventTime, eventTime+1, eventTime+2`
but because it's not the case anymore, your events will be sorted by insert order (seq id). 

-2 in this case doesn't show this, because -2 less than +1, and it's stored before +1.